### PR TITLE
Data table content loader

### PR DESCRIPTION
**Pull request by Aringoaway**
fix color for content loader

### CODE REVIEW

**Comment by Aringoaway** [comment]
@Pe5h4 can you review pls
datatable-loader

File: src/components/DataTable/index.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export function DataTable ({
 							rowClassName={customRowClassName}
 						/>}
 
-						{isLoading && contentLoader && <CellContentLoader cols={headers.length} rows={limit ?? 5} /> }
+						{isLoading && contentLoader && <CellContentLoader cols={headers.length} rows={limit ?? 5} foregroundColor={theme === "dark" ? '#788dcb' : '#fff'} /> }
```
Comment (Pe5h4) [inline src/components/DataTable/index.js:106]: @Aringoaway Is this change necessary? From the code I see if you dont specify foregroundColor, then the content loader will have the default and the default in content loader is the same as here

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Also, please add this to changelog

File: src/components/DataTable/index.js (hunk 1 above)
Comment (Aringoaway) [reply src/components/DataTable/index.js]: @Pe5h4 fixed it
